kubernetes/linux/setup.sh

```diff
@@ -10,13 +16,17 @@ sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     update-locale LANG=en_US.UTF-8
 
 #install oneagent - Official bits (10/7/2021)
-wget https://github.com/microsoft/Docker-Provider/releases/download/1.14/azure-mdsd_1.14.2-build.master.284_x86_64.deb
+if [ "${ARCH}" != "arm64" ]; then
+    wget "https://adme2epublic.blob.core.windows.net/privatefiles/azure-mdsd_1.16.0-build.dev_x86_64.deb?sp=r&st=2022-01-26T13:42:36Z&se=2022-02-24T21:42:36Z&spr=https&sv=2020-08-04&sr=b&sig=OTJREpVZsMp8WXYGpM7CMHa%2Fp22ztbWmikvmMSjX3xA%3D" -O azure-mdsd.deb
```

Comment:
Please make sure to update with official master MDSD build

Reply:
once I have access to the official master mdsd build for arm64 i plan on updating this url